docker-compose.yaml

```diff
@@ -147,22 +87,24 @@ services:
     restart: always
     image: httpd:2.4
     volumes:
-      - ${AIRFLOW_PROJ_DIR:-.}/weather:/usr/local/apache2/htdocs
-  
+      - ${AIRFLOW_PROJ_DIR}/weather:/usr/local/apache2/htdocs
+
   postgres:
     image: postgres:13
     environment:
       POSTGRES_USER: airflow
       POSTGRES_PASSWORD: airflow
       POSTGRES_DB: airflow
     volumes:
-      - postgres-db-volume:/var/lib/postgresql/data
+      - ${AIRFLOW_PROJ_DIR}/db:/var/lib/postgresql/data
```

Comment:
I'm wondering about this change to keeping the database files around and available to postgres via a bind mount instead of a docker managed volume. @johnclary, maybe we can sync for a minute on this. I'm really curious to learn about this choice.

Reply:
thanks for pointing this out @frankhereford! i completely misunderstood the way docker volumes work. i totally agree we should use a proper docker volume. i will revert this in #113.

will also update the readme to add a note about running `$ docker volume rm atd-airflow_postgres-db-volume` locally if you want to start from a scratch setup in your dev environment